code/modules/pool/pool.dm

```diff
@@ -96,6 +96,16 @@ Place a pool filter somewhere in the pool if you want people to be able to modif
 	if(do_after(user, 4 SECONDS, target = dropping))
 		splash(dropping)
 
+/turf/open/indestructible/sound/pool/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/stack/rods) && user.loc == get_step(src, NORTH) && !istype(user.loc, /turf/open/indestructible/sound/pool/))
```

Comment:
This should check to ensure that multiple pool ladders aren't being placed on the same tile. It also seems pretty restrictive to only be placed it in the solo tile above it. I think that it should simply check if the tile to the north step is not a pool tile, if so then place the ladder. I don't think it matters if someone starts building a ladder while in a pool, but if you wan't to block that then you can keep the user.loc check.

Reply:
Yeah I'll cut some of that